public/scss/_detail.scss

```diff
@@ -0,0 +1,40 @@
+$dark-blue: #505160;
+$grey: #F5F5F5;
+
+.detail {
+  line-height: 1.2em;
+  background-color: $grey;
+  width: 95%;
+  border-radius: 1em;
+  display: block;
+  margin: 0.2em auto;
+}
+.detail-title {
+  font-size: 2em;
+  text-align: center;
+  color: $dark-blue;
+  line-height: 1.5em;
+}
+.detail-image {
+  width: 5em;
+  display: block;
+  margin: 1em auto;
+}
+.detail-subtitle {
+  font-size: 1.2em;
+}
+.detail-item {
+  font-size: 1em;
+}
+.grade, .salary {
+  margin-top: -0.5em;
+}
+.last {
+  padding: 0.2em;
+}
+.line {
+  border: 0;
+  height: 1px;
```

Comment:
pixel?

http://gph.is/1jE2Wzg

Reply:
It's totally a mistake :flushed: I'm changing it!

Reply:
And the gif is great :laughing:

Reply:
@Karyum changed it :)